spec/lib/api/2.0/notification-spec.js

```diff
@@ -264,5 +264,26 @@ describe('Http.Api.Notification', function () {
                 .expect(400);
             });
         });
+
+        describe('POST /notification/alerts', function () {
+            var alert = {
+                '{"Severity":"Critical"}': "d"};
+            before(function(){
+                sinon.stub(notificationApiService, 'redfishAlertProcessing').resolves();
```

Comment:
Please restore this stub after test suites finished, so as not to bring side effect for other test cases.